### ISSUE TITLE
Doesn't work in 64-bit mode?

### ISSUE THREAD

**Issue by ExoticObjects** [Fl0p/OpenCV-iOS#4]
Could be my error...

I left the following comment [here](https://github.com/CocoaPods/Specs/issues/1058) as well. Apologies if that's bad form, but there was an awful lot of discussion about this podspec on that thread. 

<<<

I'm using the podspec successfully, to some degree - it works in 32-bit mode on the simulator, but not 64-bit. When running on 64-bit iPad or iPhone, I get errors like this:

```
Undefined symbols for architecture x86_64
```

I've been through Stackoverflow. There are lots of links. I've tried them all to no avail:

[1](http://stackoverflow.com/questions/16078512/undefined-symbols-for-architecture-x86-64-error-when-linking-opencv-in-xcode), [2](http://stackoverflow.com/questions/15070556/apple-mach-o-linker-id-error-opencv-xcode?rq=1), [3](http://stackoverflow.com/questions/18735823/opencv-link-error-in-xcode-undefined-symbols-for-architecture-x86-64-cvhaar?rq=1), [4](http://stackoverflow.com/questions/10285892/xcode-undefined-symbols-for-architecture-x86-64-error?rq=1), [5](http://stackoverflow.com/questions/12770115/linking-libraries-opencv-2-4-2-on-xcode-4-5-1?rq=1), [6](http://stackoverflow.com/questions/13564289/osx-10-7-5-opencv-2-4-3-undefined-symbols-for-architecture-x86-64-when-usin?rq=1)

Also, FWIW, OpenCV 2.4.7 is out. The last comment [here](http://richardt.name/blog/opencv-with-cxx11-on-os-x-10-8/) indicates that they've ironed out some of the issues that have made it so difficult to work with in XCode in the past. Would be awesome to have the podspec updated to use it...

**Comment by Fl0p** [comment]
Huh.. 

This pod just an builded for IOS version of the opencv repository https://github.com/Itseez/opencv.
I'm not change anything in build settings. and I'm not watching it. Sorry about this.

Looks like we have some fixes for iOS x64 - https://github.com/Itseez/opencv/commit/9631cb6de91fcc736c76e11fff6c5b6d5ff8a3ee

I'll try to make a new version at this week. In urgent case you should try to build it yourself.

**Comment by ExoticObjects** [comment]
I tried those fixes but couldn't get it to compile so that it worked for 64-bit. I then found that I could download a pre-compiled framework [here](http://sourceforge.net/projects/opencvlibrary/files/opencv-ios/2.4.7/opencv2.framework.zip/download) that _did_ work for 64-bit. So I forked your repo and added that framework. I've sent you a pull request. 

I know that you're not watching this repository too closely, but if you do `pod search openCV`, this is the only thing that comes up. So it makes sense to keep this one up to date, no?
